lib/api/objectPutCopyPart.js

```diff
@@ -310,10 +309,8 @@ function objectPutCopyPart(authInfo, request, sourceBucket,
                                     });
                                     return cb(errors.InternalError);
                                 }
-                                // skip to end of waterfall
-                                // because don't need to store
-                                // part metadata
-                                return cb(skipError);
+                                // uploadPart did the job of storing the data
+                                return cb();
```

Comment:
Don't we want to skip `getExistingPartInfo` step and `storeNewPartMetadata` steps?
I guess we are not relying on AWS anymore but on our metadata.

Reply:
correct.  we will no longer treat aws in a special manner.  it will clean up the code a lot.

Reply:
@nicolas2bert might have some incompatible changes in his azure put copy part pr.  might want to check it out.